### DISCUSSION TITLE
I would like to request the db management using the CLI

### DISCUSSION THREAD

**Discussion by Sadhguy** [auth0/auth0-cli#1211] [General]
### Checklist

- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

I need to create an application that can register different services, each one with its own users management, so I need to create a db and a connection for each service in order to achieve this goal.

### Describe the ideal solution

When a client wants to register a service in my application an auth0 application, a db and a connection between this two is automatically created so the client's service counts with it's own users management.

**Comment by willvedd** [comment]
I think I understand the ask here but how does the Auth0 CLI fit into this workflow? Are you imagining that this process would be automated upon customer request or would you be manually executing these commands through the command line?

**Comment by sergiught** [comment]
Hey @Sadhguy  👋🏻 ,

We recently added a new command 🧰 `auth0 api <method> <url-path> [flags]` that is able to make authenticated requests against the management API (https://auth0.com/docs/api/management/v2). 

:octocat: This command is available with https://github.com/auth0/auth0-cli/releases/tag/v0.13.0. 

📖 Docs for this command: https://auth0.github.io/auth0-cli/auth0_api.html

Although it doesn't offer first class support for managing database connections, you could use it as an alternative to manage database connections through the cli until we have a more dedicated command for them.

For example to create a database connection: `auth0 api post connections  --data "{\"strategy\":\"auth0\", etc...}"`

Feedback is much appreciate for this new command 🙏🏻 Let us know if you encounter any issues with it. 

We'll keep this issue open however until we are able to add a dedicated first class command to manage db connections. 

Thanks for your patience! ❤️
